Formula/n/npm-check-updates.rb

```diff
@@ -0,0 +1,40 @@
+class NpmCheckUpdates < Formula
+  require "language/node"
```

Comment:
also needs can be moved to the first line to consistent with other nodejs formulae